Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.16.5-alpine
```

Comment:
Why not go v1.17, which we use for all other projects?

Reply:
Updated